main.c

```diff
@@ -534,6 +537,10 @@ STATIC int run_repl(void) {
     return exit_code;
 }
 
+#ifdef MY_DEBUGUART
```

Comment:
Instead of checking `MY_DEBUGUART`, could you name it `NRF_DEBUG_PRINT` flag or similar?